init.lua

```diff
@@ -222,3 +222,31 @@ minetest.register_chatcommand("whereis", {
 				pos.x, pos.y, pos.z)
 	end
 })
+
+local staff = {}
+minetest.register_chatcommand("s", {
+	params = "<msg>",
+	description = "Send a message on the staff channel",
+	privs = { kick = true, ban = true },
+	func = function(name, param)
+		for _, toname in pairs(staff) do
+			minetest.chat_send_player(toname, minetest.colorize("#ff9900",
+				"«" .. name .. "» " .. param))
```

Comment:
I think it might be better to avoid non-ASCII signs here.

Reply:
Agreed